internal/core/algorithm/ngt/ngt.go

```diff
@@ -156,7 +156,7 @@ const (
 	// -------------------------------------------------------------
 	// dimension constraints
 	// -------------------------------------------------------------.
-	ngtVectorDimensionSizeLimit = 1 << 16
+	NgtVectorDimensionSizeLimit = 1<<32 - 1
```

Comment:
for public variable it would be better to prevent redundant naming like ngt.NgtVectorDimensionSizeLimit

I think it's better to name as ngt.VectorDImensionSizeLimit